canvas_sdk/v1/data/observation.py

```diff
@@ -0,0 +1,117 @@
+from django.db import models
+
+from canvas_sdk.v1.data.base import CommittableModelManager, ValueSetLookupQuerySet
+from canvas_sdk.v1.data.patient import Patient
+from canvas_sdk.v1.data.user import CanvasUser
+
+
+class ObservationQuerySet(ValueSetLookupQuerySet):
+    """ObservationQuerySet."""
+
+    pass
+
+
+class Observation(models.Model):
+    """Observation."""
+
+    class Meta:
+        managed = False
+        app_label = "canvas_sdk"
+        db_table = "canvas_sdk_data_api_observation_001"
+
+    objects = CommittableModelManager.from_queryset(ObservationQuerySet)()
+
+    id = models.UUIDField()
+    dbid = models.BigIntegerField(primary_key=True)
+    created = models.DateTimeField()
+    modified = models.DateTimeField()
+    originator = models.ForeignKey(CanvasUser, on_delete=models.DO_NOTHING)
```

Comment:
Just happened to stumble across this commit and saw all these `DO_NOTHING`. 
They are usually a bad thing, so wanted to ask why the reasoning behind it?

Reply:
Honestly I wonder if they're needed at all. The models are backed by views, and the db connection is read only, so there's no possibility of deletion. I think `DO_NOTHING` just avoids django from even trying.

Reply:
`on_delete` is a required argument for `ForeignKey`, so `DO_NOTHING` is probably the most appropriate value for read-only unmanaged models that are backed by views.